config/htcondor-ce.spec

```diff
@@ -277,10 +277,10 @@ fi
 %defattr(-,root,root,-)
 
 # Web package
-%{python_sitelib}/htcondor_ce
-%{python_sitelib}/htcondor_ce/__init__.py
-%{python_sitelib}/htcondor_ce/web.py
-%{python_sitelib}/htcondor_ce/rrd.py
+%{python_sitelib}/htcondorce
+%{python_sitelib}/htcondorce/__init__.py*
+%{python_sitelib}/htcondorce/web.py*
+%{python_sitelib}/htcondorce/rrd.py*
```

Comment:
Why are you including the * after the .py? Rpmbuild will automatically add the pyc when it precompiles the python files.